package.json

```
@@ -27,6 +27,7 @@
   "main": "lib/cjs/index.js",
   "module": "lib/esm/index.js",
   "types": "lib/types/index.d.ts",
+  "sideEffects": ["./src/generators/*FileGenerator.ts", "./src/helpers/FileHelpers.ts"],
```

We read in webpack docs:

> All the code noted above does not contain side effects, so we can mark the property as false to inform webpack that it can safely prune unused exports.

so, we should change it to:

```suggestion
  "sideEffects": false
```

`"./src/generators/*FileGenerator.ts", "./src/helpers/FileHelpers.ts"]` haven't any side effects for other code in package - these files only use another code and aren't used in other place :)